core/google/cloud/connection.py

```diff
@@ -273,7 +273,7 @@ def _do_request(self, method, url, headers, data,
                                  body=data)
 
     def api_request(self, method, path, query_params=None,
-                    data=None, content_type=None,
+                    data=None, content_type=None, headers=None,
```

Comment:
Are there a lot of different headers that could be passed?

My thought is that if we have encryption headers on other APIs the it might be nice to just have those setup automatically and pass in the values to populate them.